### ISSUE TITLE
Jost font problem with letters Ū and ū

### ISSUE THREAD

**Issue by sundian** [google/fonts#7017]
**Describe the bug**
Jost font issue. The stroke above the letter Ū and ū appears to have shifted to the right.

- Letter Ū code `U+016A` html code `&Umacr;` description: Latin Capital Letter U with macron
- Letter ū code `U+016B` html code `&umacr;` description: Latin Small Letter U with macron

**To Reproduce**
Problem appears when letter Ū and ū are after other letters. When letter comes first in paragraph it looks fine. Issue appears in Chrome browsers.

**Screenshots**
![Jost font issue](https://github.com/google/fonts/assets/3251276/be37528a-e437-4ec1-8bf7-d36f023e80e7)

**Additional context**
It appears on Chrome browsers (pc and android). It looks fine in other browsers (eg Firefox).

**Comment by RosaWagner** [comment]
Hi @sundian, thanks for the report. The font may need to be updated with better  base_to_mark support. It is added to the pipeline.

**Comment by RosaWagner** [comment]
As a note for next year: I see the family has proper anchor placement etc, so it might be a subsetter-only issue. Cf video below:

https://github.com/google/fonts/assets/12222436/443b146c-df84-43ee-9812-06ff5a616923

**Comment by r-shestilovski** [comment]
Any ETA on this?

**Comment by bghryct** [comment]
@EwonRael maybe you and I can take a look at this soon?

**Comment by katmai7** [comment]
Looks like this is related to: https://github.com/google/fonts/issues/7904
Example: https://jsfiddle.net/hysnrqge/4/

**Comment by chrissimpkins** [comment]
@emmamarichal do you the capacity to take on just the bug fix as described in the initial report here, ignoring the fontbakery check issues?

**Comment by emmamarichal** [comment]
@chrissimpkins Status: still in progress on my side

**Comment by JPM9623** [comment]
Hi guys, I notice this issue is specifically about the characters Ū and ū. I opened another issue which was closed as it is technically a duplicate of this issue, but I just wanted to note that this is affecting all vowels, lower and upper case. Apologies if this has already been clarified.

**Comment by emmamarichal** [comment]
Hi @JPM9623! Yes, I'm working on it currently and I fixed this issue for all vowels. I need to build the fonts again and we should be good. But the process can take a bit of time since we have some other deadlines aside. Thank you for your patience!

**Comment by Domaso999** [comment]
Hi, 4 months have passed since the last comment - just wondering if the Jost font "ū" issue will be fixed soon?

**Comment by sundian** [comment]
Issue wiht  'ū' still appear. In google fonts page [https://fonts.google.com/specimen/Jost?preview.text=Strukt%C5%ABra](url)  seems OK, but if you click on some preview line issue apears. Also tried on my page, still apear (with browser cache clear).
Before click:
![Image](https://github.com/user-attachments/assets/edb081d1-f9a2-483a-a8ed-08db1f8bff19)

After click:
![Image](https://github.com/user-attachments/assets/cd80711a-723c-4fbc-b21b-08ef0daa7225)

On real page:
![Image](https://github.com/user-attachments/assets/ddb1f77c-a99d-44f6-8c63-bd8325dbffe3)

**Comment by sundian** [comment]
> Hi, 4 months have passed since the last comment - just wondering if the Jost font "ū" issue will be fixed soon? Thanks.

Issue still appear, not fixed now. You are Domas from LT? If yes it's funny, because I'm Domas to.

**Comment by Domaso999** [comment]
> > Hi, 4 months have passed since the last comment - just wondering if the Jost font "ū" issue will be fixed soon? Thanks.
> 
> Issue still appear, not fixed now. You are Domas from LT? If yes it's funny, because I'm Domas to.

Yep, also Domas from Lithuania :) Can't wait for Jost font to be fixed... Such a nice font, I like to use it in website designs, it's a pitty we are having the "ū" problem unfixed for such a long time now...

**Comment by garretrieger** [comment]
Looked into this and I believe it's a problem in the font combined with how we do our subsetting. The Jost font defines support for the U and 0x0304 combining macron code point and has a uni0304 glyph for 0x0304. However, it does not define either a GSUB substitution nor GPOS rule for handling U + 0x0304.

Ideally you'd have at least one of the following:
1. A substitution which converts U + 0x0304 into the U macron glyph
2. A positioning rule which for U + 0x0304 correctly positions the macron glyph above the U

For some context here's what goes wrong:
1. Our core latin definition contains  U and 0x0304 but not 0x016a.
2. Our latin ext subset contains the 0x016A.

When a 0x016a codepoint is encountered in text the browser decides that since it can be decomposed to U + 0x0304 that the core latin file does support this code point and attempts to render it using core latin font and U + 0x0304. However, due to the absence of either a substitution or positioning rule to make that render correctly.

With the unsubsetted font it doesn't look like there's an issue since shapers will automatically compose U + 0x0304 into 0x016a which will result in the Umacron glyph.

In summary if the font is fixed as I suggested above then this should start rendering correctly with the Google Fonts subsets.

**Comment by Emavu** [comment]
Will there be a solution to this? I can't wait :)
